.gitignore

```diff
@@ -5,24 +5,15 @@ nbproject
 .settings
 .project
 config/sync/.htaccess
-config/sync/README.txt
+#config/sync/README.txt
 docker/docker-compose.override.yml
 node_modules
 vendor
-web/.csslintrc
-web/.editorconfig
-web/.eslintignore
-web/.eslintrc.json
-web/.gitattributes
-web/.htaccess
-web/autoload.php
+web/*.*
 web/core/
-web/index.php
 web/libraries/
 web/modules/contrib/
 web/profiles/contrib/config_installer/
-web/robots.txt
 web/sites/
 web/themes/contrib/
-web/update.php
-web/web.config
+mysql/
```

Comment:
maybe `.data/` for other mounts like mysql

Reply:
Its example folder, added small explanation in `.env.default`